src/DynamoCore/Nodes/NodeCategories.cs

```diff
@@ -411,6 +496,57 @@ internal static string MakeAbsolutePath(string basePath, string relativePath)
             return resultUri.LocalPath;
         }
 
+        /// <summary>
+        /// Add spaces to string before capital letters e.g. CoordinateSystem to Coordinate System.
+        /// </summary>
+        /// <param name="original">incoming string</param>
```

Comment:
Same [internal task](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6125) for adding unit test cases for `InsertSpacesToString` method.